octobot_trading/enums.py

```diff
@@ -381,9 +387,22 @@ class ExchangeConstantsOrderColumns(enum.Enum):
     REDUCE_ONLY = "reduceOnly"
     STOP_PRICE = "stopPrice"
     TRIGGER_ABOVE = "triggerAbove"
+    POST_ONLY = "post_only"
     TAG = "tag"
 
 
+class CryptoFeedOrderColumns(enum.Enum):
```

Comment:
cryptofeed order is an object
we convert it into a dict so that it works with the default orders parser (less to maintain)

for example:
order.status gets converted to order_dict[CryptoFeedOrderColumns.STATUS.value]
see cryptofeed_orders_parser 

or do you mean we should move the enum to cryptofeed parser?

Reply:
I removed it, its not necessary